docs/user/customization.md

```diff
@@ -449,6 +449,72 @@ Example application of `loglevel=7` (change Linux kernel log level to KERN_DEBUG
 
     If you wish to confirm the kernel argument is indeed being applied on the system, you can `oc debug` into a node and check with `rpm-ostree kargs`.
 
+#### Switching RHCOS host kernel using KernelType
+
+With OCP 4.4 and onward release, it is possible to switch from traditional to Real Time (RT) kernel on RHCOS node. During install time, switching to RT kernel can be done through manifests as an installer operation. See [customizing MachineConfig](https://github.com/openshift/installer/blob/ee1b793414178173e03c55b0e6c891849777531c/docs/user/customization.md#install-time-customization-for-machine-configuration) to configure kernelType during install time. To set kernelType as day 2 operation, see [MachineConfiguration](https://github.com/openshift/machine-config-operator/blob/master/docs/MachineConfiguration.md#kernelType) doc.
+
+Example for switching to RT kernel on worker nodes during initial cluster install:
+
+1. Run `manifests` target to create all the manifests.
+
+    ```console
+    $ mkdir realtime_kernel
+
+    $ openshift-install --dir realtime_kernel create manifests
+    ...
+
+    $ ls -l realtime_kernel/openshift/
+    ...
+    99_openshift-machineconfig_99-master-ssh.yaml
+    99_openshift-machineconfig_99-worker-ssh.yaml
+    ...
+    ```
+
+2. Create a `MachineConfig` that sets `kernelType` to `realtime`:
+
+    ```sh
+    cat > realtime_kernel/openshift/99-worker-kerneltype.yaml <<EOF
+    apiVersion: machineconfiguration.openshift.io/v1
+    kind: MachineConfig
+    metadata:
+      labels:
+        machineconfiguration.openshift.io/role: "worker"
+      name: 99-worker-kerneltype
+    spec:
+      kernelType: realtime
+    EOF
+    ```
+
+3. Run `cluster` target to create the cluster using the custom manifests.
+
+    ```console
+    $ openshift-install --dir realtime_kernel create cluster
+    ...
+    ```
+
+    Check that the MachineConfig has the kernelType applied
+
+    ```console
+    $ oc --config realtime_kernel/auth/kubeconfig get machineconfigs
+
+    NAME                                                        GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
+    ...
+    99-worker-kerneltype                                                                                                     80m
+    99-worker-ssh                                                                                          2.2.0             80m
+    rendered-worker-853ba9bf0337db528a857a9c7380b95a            6306be9274cd3052f5075c81fa447c7895b7b9f4   2.2.0             78m
+    ...
+
+4. To confirm that worker node has switched to RT kernel, access one of the worker node and run `uname -a`
+
+```console
+$ oc --config realtime_kernel/auth/kubeconfig debug node/<worker_node>
```

Comment:
nit: your other console examples are indented by four spaces so they are rendered as part of their associated enumerated list item.  Can we indent this block as well?  Otherwise it will appear after the enumerated list closes.

Reply:
yeah, thanks for noticing. fixed